### PR TITLE
[CROSSDATA-495] Multiple logger implementations are being packaged

### PR DESCRIPTION
**Pull request by jjlopezm**
## Description
This PR fixes the multiples slf4j that have been server and driver.

### Testing
- [ ] Test in a docker

### Documentation
- [ ] Changelog update
- [ ] https://stratio.atlassian.net/wiki/display/CROSSDATA/Get+Started

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Only listing significant user-visible, not internal code cleanups and minor bug
 * Improved communication between client and server.
 * Native jobs available in SparkUI
 * Improved native access to MongoDB
+* Fix multiples implementation of Slf4j
```
Comment (miguel0afd) [inline CHANGELOG.md]: I would say that it wasn't a major issue, so I wouldn't include it in the Changelog

**Comment by miguel0afd** [comment]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (darroyocazorla) [inline CHANGELOG.md]: agree

**Comment by darroyocazorla** [comment]
+1

File: CHANGELOG.md (hunk 1 above)
Comment (jjlopezm) [inline CHANGELOG.md]: But the bug is set to Major, mmm, I change it too

**Pull request merged**